addons/binding/org.openhab.binding.robonect/pom.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://maven.apache.org/POM/4.0.0"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>org.openhab.binding</groupId>
+        <artifactId>pom</artifactId>
+        <version>2.1.0-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>org.openhab.binding.robonect</artifactId>
+    <version>2.1.0-SNAPSHOT</version>
+
+    <name>Robonect Binding</name>
+    <packaging>eclipse-plugin</packaging>
+
+    <dependencies>
```

Comment:
I'm almost done locally with this refactoring for the tests... However, I stuck with hamcrest...

If I use in MANIFEST.MF
`Require-Bundle: org.hamcrest`

everything works ok but PMD/Checkstyle complains with

` ERROR found by checkstyle: .binding.robonect.test/META-INF/MANIFEST.MF:21 The MANIFEST.MF file must not contain any Require-Bundle entries. Instead, Import-Package must be used.`

If I remove the `Require-Bundle` part just leaving the following `Import-Package` declaration:

`
Import-Package: org.openhab.binding.robonect,
 org.openhab.binding.robonect.handler,
 org.openhab.binding.robonect.model,
 org.eclipse.smarthome.core.common.registry,
 org.eclipse.smarthome.core.events,
 org.eclipse.smarthome.core.thing.util,
 org.eclipse.smarthome.test.java,
 org.eclipse.smarthome.test.storage,
 org.mockito,
 org.mockito.invocation,
 org.mockito.stubbing,
 org.mockito.verification,
 org.hamcrest;core=split,
 org.junit;version="4.0.0",
 org.junit.runner,
 org.junit.runners,
 org.osgi.framework,
 org.osgi.service.device,
 org.slf4j,
 org.eclipse.smarthome.core.thing,
 org.eclipse.smarthome.core.types,
 org.eclipse.smarthome.core.library.types,
 org.eclipse.jetty.client,
 org.eclipse.jetty.http
`

The tests are not executed via maven anymore resulting in 

```
!ENTRY org.mockito 4 0 2017-06-29 09:17:00.889
!MESSAGE FrameworkEvent ERROR
!STACK 0
org.osgi.framework.BundleException: Could not resolve module: org.mockito [34]
  Unresolved requirement: Import-Package: COM.jrockit.reflect; resolution:="optional"
  Unresolved requirement: Import-Package: jrockit.vm; resolution:="optional"
  Unresolved requirement: Import-Package: org.hamcrest; version="[1.0.0,2.0.0)"

	at org.eclipse.osgi.container.Module.start(Module.java:434)
	at org.eclipse.osgi.container.ModuleContainer$ContainerStartLevel.incStartLevel(ModuleContainer.java:1582)
	at org.eclipse.osgi.container.ModuleContainer$ContainerStartLevel.incStartLevel(ModuleContainer.java:1561)
	at org.eclipse.osgi.container.ModuleContainer$ContainerStartLevel.doContainerStartLevel(ModuleContainer.java:1533)
	at org.eclipse.osgi.container.ModuleContainer$ContainerStartLevel.dispatchEvent(ModuleContainer.java:1476)
	at org.eclipse.osgi.container.ModuleContainer$ContainerStartLevel.dispatchEvent(ModuleContainer.java:1)
	at org.eclipse.osgi.framework.eventmgr.EventManager.dispatchEvent(EventManager.java:230)
	at org.eclipse.osgi.framework.eventmgr.EventManager$EventThread.run(EventManager.java:340)

!ENTRY org.openhab.binding.robonect.test 2 0 2017-06-29 09:17:01.079
!MESSAGE Could not resolve module: org.openhab.binding.robonect.test [37]
  Unresolved requirement: Import-Package: org.mockito
    -> Export-Package: org.mockito; bundle-symbolic-name="org.mockito"; bundle-version="1.9.5.v201311280930"; version="1.9.5"; uses:="org.mockito.stubbing,  org.hamcrest,  org.mockito.verification,  org.mockito.listeners,  org.mockito.exceptions.base,  org.mockito.exceptions,  org.mockito.configuration,  org.mockito.invocation"
       org.mockito [34]
         Unresolved requirement: Import-Package: COM.jrockit.reflect; resolution:="optional"
         Unresolved requirement: Import-Package: jrockit.vm; resolution:="optional"
         Unresolved requirement: Import-Package: org.hamcrest; version="[1.0.0,2.0.0)"


!ENTRY org.mockito 2 0 2017-06-29 09:17:01.079
!MESSAGE Could not resolve module: org.mockito [34]
  Unresolved requirement: Import-Package: COM.jrockit.reflect; resolution:="optional"
  Unresolved requirement: Import-Package: jrockit.vm; resolution:="optional"
  Unresolved requirement: Import-Package: org.hamcrest; version="[1.0.0,2.0.0)"
```

Any suggestions or samples on how to use moskito/hamcrest via pure imports? 
Also looking at the other bindings I could not find any not having at least hamcrest defined via Require-Bundle. So how can they skip this check style rule?